### PR TITLE
Add yadda part 1

### PR DESCRIPTION
**Pull request by sriking1783**
Yadda part 1

### CODE REVIEW

File: yadda/create_yadda_schema.sql [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  created_by   TEXT         ,
+  created_on   TIMESTAMPTZ  ,
+  updated_by    TEXT         ,
+  updated_on    TIMESTAMPTZ
+);
+
+CREATE TABLE yadda.breweries(
+  id            SERIAL       PRIMARY KEY,
+  name          TEXT         NOT NULL,
+  address_id    INTEGER      NOT NULL REFERENCES yadda.addresses,
```
Comment (incazteca) [inline yadda/create_yadda_schema.sql]: You should have something to handle what happens to the relation upon deletion.

File: yadda/create_yadda_schema.sql [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  created_by   TEXT         ,
+  created_on   TIMESTAMPTZ  ,
+  updated_by    TEXT         ,
+  updated_on    TIMESTAMPTZ  ,
+  description   TEXT
+  );
+
+CREATE TABLE yadda.beers(
+  id            SERIAL       PRIMARY KEY,
+  style         TEXT         NOT NULL,
```
Comment (incazteca) [inline yadda/create_yadda_schema.sql]: Consider normalizing this column

File: yadda/create_yadda_schema.sql [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  brewing_year  INTEGER      NOT NULL,
+  brewery_id    INTEGER      REFERENCES yadda.breweries,
+  created_by   TEXT         ,
+  created_on   TIMESTAMPTZ  ,
+  updated_by    TEXT         ,
+  updated_on    TIMESTAMPTZ  ,
+  description   TEXT
+);
+
+CREATE TABLE yadda.users(
```
Comment (incazteca) [inline yadda/create_yadda_schema.sql]: Consider giving the user someplace to live.

File: yadda/create_yadda_schema.sql [hunk 4]
```diff
@@ -0,0 +1,160 @@
+BEGIN;
+
```
Comment (incazteca) [inline yadda/create_yadda_schema.sql]: Drop tables if they already exist.

File: yadda/create_yadda_schema.sql [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  updated_by    TEXT         ,
+  updated_on    TIMESTAMPTZ
+);
+
+
+CREATE OR REPLACE FUNCTION  yadda.tg_inserts()
+RETURNS TRIGGER AS $$
+  BEGIN
+    NEW.created_on := current_timestamp;
+    NEW.created_by := current_user;
```
Comment (incazteca) [inline yadda/create_yadda_schema.sql]: Give the  `updated_by`, `updated_on` fields an initial value when the row is created.

**Comment by incazteca** [comment]
@sriking1783 Please add the proper cascades. Also the beer styles can be put in their own lookup table. Please submit with fixes.

**Comment by djkotowski** [comment]
Looking good.  Take a look at the comments, make changes and resubmit.

File: yadda/create_yadda_schema.sql (hunk 5 above)
Comment (sriking1783) [inline yadda/create_yadda_schema.sql]: @incazteca  My intention was populate the created_* fields when inserted.
Populate updated_* results when updated.
Do you really think it makes sense to populate updated_* columns even though we are not updating?

**Pull request closed without merging**